_includes/manuals/nightly/3.4_install_from_source.md

```diff
@@ -4,9 +4,9 @@ Foreman uses Bundler to install dependencies and get up and running. This is the
 
 Foreman will run with the following requirements (aside from rubygem dependencies):
 
-* Ruby 2.5 or newer
-* NodeJS 10 or newer
-* NPM 3.0 or newer
+* Ruby 2.7.x
+* NodeJS 12 or 14
+* NPM 6.0 or newer
```

Comment:
We dont support npm 7+ (reasons described here: https://community.theforeman.org/t/javascript-infrastructure-update-plan/28614)

Reply:
Thanks, so I will change it to `NPM 6.x`.